tests/integration/test_relationships.py

```diff
@@ -1,46 +0,0 @@
-import pytest
```

Comment:
why was this file deleted?

Reply:
this test was designed specifically to test `Relationship` between project and dataset but this does not exist for Batch projects... I think equivalent would be to use projects.datasets.connect(dataset) and this is tested indirectly in tests/integration/test_sorting.py.. but actually let me ask @kkim-labelbox to verify this test can be remove